thirdparty/fluid/modules/gapi/include/opencv2/gapi/util/variant.hpp.patch

```diff
@@ -0,0 +1,5 @@
+512,513d511
+< #pragma GCC diagnostic push
+< #pragma GCC diagnostic ignored "-Wmaybe-uninitialized"
+528d525
+< #pragma GCC diagnostic pop
```

Comment:
Does fluid itself report the same problem?

Reply:
@riverlijunjie , No, remove it, patch is not a good way, I think.